doc/build-freebsd.md

```diff
@@ -36,11 +36,11 @@ pkg install sqlite3
 ```
 
 ###### Legacy Wallet Support
-`db5` is only required to support legacy wallets.
+`db4` is only required to support legacy wallets.
 Skip if you don't intend to use legacy wallets.
 
 ```bash
-pkg install db5
+./contrib/install_db4.sh `pwd`
```

Comment:
Some users read the docs from master, even if they compile a previous release. So they'd run into issues, due to https://github.com/bitcoin/bitcoin/pull/26772

Reply:
Thanks for commenting. I can see how building a previous release using `master` docs as a guide may result in running into issues.  In that case, hopefully a search for the error would find #26774.  But as mentioned here https://github.com/bitcoin/bitcoin/blob/65de8eeeca29e71378aa34602b287ab921b040e4/doc/build-unix.md?plain=1#L68 and here https://github.com/bitcoin/bitcoin/blob/65de8eeeca29e71378aa34602b287ab921b040e4/doc/build-openbsd.md?plain=1#L43 building with db5 will break compatibility with legacy wallets which is more problematic. #26772 aside, this doc should probably be updated.

Reply:
This shouldn't be an issue, see https://github.com/bitcoin/bitcoin/pull/18870

Reply:
My takeaway from #18870 is that it supports this PR. As @achow101 stated when he closed it:

        This seems to still be a bit problematic because of those transaction logs. So closing for now.


It's entirely possible I'm missing something, but advising someone needing legacy wallet support to build with db5 seems to go down the wrong path and build docs for platforms other than FreeBSD address this. Do you think adding more rationale in the document would be helpful?

Reply:
yeah, my understanding is that you don't have logs in normal operation, so this shouldn't be an issue in normal operation. And not normal operation isn't supported by this software project.

No strong opinion either way. I think anything is fine here, I just left a comment.